samcli/cli/context.py

```diff
@@ -50,9 +55,12 @@ def debug(self, value):
         self._debug = value
 
         if self._debug:
-            # Turn on debug logging
-            logging.getLogger("samcli").setLevel(logging.DEBUG)
-            logging.getLogger("aws_lambda_builders").setLevel(logging.DEBUG)
+            # Turn on debug logging and display timestamps
+            for logger_name in [SAM_CLI_LOGGER_NAME, LAMBDA_BULDERS_LOGGER_NAME]:
```

Comment:
Just noticing this. Why are we setting the logger directly instead of through `SamCliLogger.configure_logger` like we do in other places?

Reply:
`SamCliLogger.configure_logger` will add a new handler. We should change the existing handlers here.

Reply:
But I do think we should pack this into `SamCliLogger` though!

Reply:
(done)